index2.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Note-taking App</title>
+</head>
+<body>
+  <div class="container">
+    <h1>Note-taking App</h1>
+    <form id="note-form">
+      <input type="text" id="note-title" placeholder="Note Title">
+      <textarea id="note-content" placeholder="Note Content">
+      <button type="submit" id="add-note">Add Note</button>
```

Comment:
Both buttons have type='submit' which will cause the form to be submitted when either button is pressed. Consider changing the type of the 'Update Note' button to 'button' if it is not intended to submit the form.